src/main/java/robot/Constants.java

```diff
@@ -20,22 +20,24 @@
 
     public static class Shooter {
         public static final int TALON_PID_SLOT = 0;
-        public static final double KP = 0;
+        public static final double KP = 0.5;
         public static final double KI = 0;
         public static final double KD = 0;
         public static final double KF = 0;
-        public static final int TIMEOUT_MS= 0;
-        public static final boolean IS_MASTER_INVERTED = false;
-        public static final boolean IS_SLAVE_INVERTED = false;
+        public static final int TIMEOUT_MS = 20;
```

Comment:
Why did you change this? IIRC, we've always used 10ms, quite arbitrarily I think…